lhapdf.sh

```diff
@@ -3,12 +3,19 @@ version: v6.1.5
 source: https://github.com/alisw/LHAPDF
 tag: v6.1.5
 requires:
-  - yaml-cpp
+ - yaml-cpp
+ - boost
+ - "autotools:(slc5).*"
+force_rebuild: true
 ---
 #!/bin/sh
 # Does not work out of source.
 
 cd $SOURCEDIR
+
+# autoreconf if using our own build of autotools
+[[ -n ${AUTOTOOLS_ROOT} ]] && autoreconf -ivf
```

Comment:
This will exit with 1 if ${AUTOTOOLS_ROOT} is not defined, no? We run recipes with /bin/bash -e so I'm not sure how this can work.